opg-kibana-1604/Dockerfile

```diff
@@ -4,19 +4,19 @@ ENV KIBANA_VERSION 5.4.0
 ENV ES_SKIP_SET_KERNEL_PARAMETERS true
 
 # add our user and group first to make sure their IDs get assigned consistently
-RUN groupadd -r kibana && useradd -r -g kibana kibana
+RUN groupadd -r kibana && \
+    useradd -r -g kibana kibana && \
+    apt-get update && \
+    apt-get install -y ca-certificates curl && \
```

Comment:
Could installation of curl and CA be part of the next step, since it's that step that requires it